README.md

```diff
@@ -96,6 +96,39 @@ export OPENAI_API_KEY='your-openai-api-key'
 
 ## 🎬 Quickstart
 
+1. Define agent reasoning workflow in `workflow.yml`:
```

Comment:
I'm not sure why this should be the main entry point into Adala and even be present in the readme. To me that looks like a possible addition worth speaking about in the docs.

Reply:
It's a tentative to get a more human-readable form of creating agents. We can remove it from readme, if it doesn't bring too much value, however it might be more convenient and more robust rather than copypasting and changing the python code (based on my experience of creating long reasoning chains)